layouts/partials/footer.html

```diff
@@ -13,6 +13,7 @@
         <!-- /.col-md-4 -->
         {{ end }}
 
+        {{ if isset .Site.Params "recent_posts" }} {{ if .Site.Params.recent_posts.enable }} 
         <div class="col-md-4 col-sm-6">
```

Comment:
@josesalvador Thanks for the PR. Can you move this <div> out of the double `if`? This way we keep the Contact block size unaltered independently of whether Recent Posts are enabled or disabled.